application/aam-backend-service/src/main/kotlin/com/aamdigital/aambackendservice/changes/core/CouchDatabaseChangeDetection.kt

```diff
@@ -0,0 +1,94 @@
+package com.aamdigital.aambackendservice.changes.core
+
+import com.aamdigital.aambackendservice.changes.core.event.DatabaseChangeEvent
+import com.aamdigital.aambackendservice.changes.di.ChangesQueueConfiguration.Companion.DB_CHANGES_QUEUE
+import com.aamdigital.aambackendservice.changes.repository.SyncEntry
+import com.aamdigital.aambackendservice.changes.repository.SyncRepository
+import com.aamdigital.aambackendservice.couchdb.core.CouchDbStorage
+import com.aamdigital.aambackendservice.couchdb.core.getEmptyQueryParams
+import org.slf4j.LoggerFactory
+import reactor.core.publisher.Mono
+import java.util.*
+
+class CouchDatabaseChangeDetection(
```

Comment:
otherwise always using CouchDb instead of CouchDatabase

Reply:
Should be "CouchDbDatatbaseChangeDetection